flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/NettyShuffleEnvironmentConfigurationTest.java

```diff
@@ -76,4 +80,25 @@ public void testNetworkRequestBackoffAndBuffers() {
         assertEquals(networkConfig.networkBuffersPerChannel(), 10);
         assertEquals(networkConfig.floatingNetworkBuffersPerGate(), 100);
     }
+
```

Comment:
I think the test is not related to the issue. So maybe we could open another issue for this. WDYT?